### PR TITLE
Updated Jenkins, JFR & plugins

### PR DESCRIPTION
**Pull request by alxsap**

### CODE REVIEW

**Comment by alxsap** [comment]
Image available for testing: `stewardci/stewardci-jenkinsfile-runner:TEST_200526_71d9927`

File: update/generate.groovy [hunk 1]
```diff
@@ -4,7 +4,7 @@ import groovy.json.JsonOutput
 wantedPlugins = [:]
 resultingPlugins = [:]
 updateCenter = null
-debug = false
+debug = true
```
Comment (romanisb) [inline update/generate.groovy:7]: Do you want to keep debug on?

Comment (alxsap) [reply update/generate.groovy:7]: I'll check. I think it is more "verbose" than "debug"

**Comment by alxsap** [comment]
New image for testing: `stewardci/stewardci-jenkinsfile-runner:TEST_200528_b5ee2e5`

**Comment by alxsap** [comment]
Reference Pipeline successful w/o checkmarx plugin

File: update.sh [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
+set -eu -o pipefail
 
-printf "Updating plugins\n"
+printf "### Updating plugins ###########\n"
 ./update/updatePlugins.sh
 
-printf "\nUpdating Jenkins LTS version\n"
-printf "(not automated yet - update packager-config.yml manually)\n"
+printf "\n\n### Updating Jenkins LTS version ###########\n"
+./update/updateJenkins.sh
```
Comment (romanisb) [inline update.sh:8]: Newline missing here.

File: update/generate.groovy (hunk 1 above)
Comment (alxsap) [reply update/generate.groovy]: changed to "verbose", kept true

**Comment by alxsap** [comment]
Image TEST_200529_15faf21 available

**Comment by alxsap** [comment]
b8fdcff verified successfully

**Comment by alxsap** [comment]
15faf21 validation failed. `warnings-ng` plugin is required for reference pipeline.

**Comment by alxsap** [comment]
c721bf6 validation failed. `badge` plugin missing for reference pipeline

**Comment by alxsap** [comment]
835397a seems to work. Waiting for full validation

File: jenkinsfile-runner-base-image/plugins.txt [hunk 3]
```diff
@@ -19,15 +18,12 @@ influxdb
...
 kubernetes-credentials-provider
-nexus-artifact-uploader
 pipeline-github
 pipeline-github-lib
 pipeline-utility-steps
 pmd
-ssh-agent
-subversion
 timestamper
+workflow-aggregator
```
Comment (alxsap) [inline jenkinsfile-runner-base-image/plugins.txt:26]: why this move?

**Comment by alxsap** [comment]
Reference pipelines successful

**Pull request merged**